DesignSafe-CI/portal
# DES-1920: One License per DOI

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [DES-1920](https://jira.tacc.utexas.edu/browse/DES-1920)

## Summary of Changes: ##
Updated the radio buttons to select a license to allow users to only select one at a time.

## Testing Steps: ##
1. Run a project through the publication pipeline to the last step
2. Make sure it doesn't allow you to select multiple licenses on the page.

## UI Photos:

## Notes: ##
